src/server/game/DataStores/DB2Stores.cpp

```diff
@@ -837,6 +837,12 @@ void DB2Manager::LoadStores(std::string const& dataPath, uint32 defaultLocale)
     for (AzeriteUnlockMappingEntry const* azeriteUnlockMapping : sAzeriteUnlockMappingStore)
         azeriteUnlockMappings[azeriteUnlockMapping->AzeriteUnlockMappingSetID].push_back(azeriteUnlockMapping);
 
+    for (BattlemasterListEntry const* battlemaster : sBattlemasterListStore)
+    {
+        ASSERT(battlemaster->MaxLevel >= battlemaster->MinLevel, "Battlemaster (%u) contains bad values for MinLevel (%u) and MaxLevel (%u)", battlemaster->ID, battlemaster->MinLevel, battlemaster->MaxLevel);
+        ASSERT(battlemaster->MaxPlayers >= battlemaster->MinPlayers, "Battlemaster (%u) contains bad values for MinPlayers (%u) and MaxPlayers (%u)", battlemaster->ID, battlemaster->MinPlayers, battlemaster->MaxPlayers);
```

Comment:
never assert on db2 data like this, the only time it needs to be done if some compile time constant depends on it like array sizes, this case here should be handled like sGameObjectDisplayInfoStore

Reply:
Done! :)